packages/node_modules/nav-frontend-toggle-style/README.md

```diff
@@ -3,7 +3,7 @@
 <!-- AUTO-GENERATED-CONTENT:START (INSTALL) -->
 ### Installering:
 ```
-npm install nav-frontend-toggle-style nav-frontend-core nav-frontend-typografi-style nav-frontend-knapper-style react --save
+npm install nav-frontend-toggle-style nav-frontend-core nav-frontend-knapper-style nav-frontend-typografi-style react --save
```

Comment:
Disse burde vel egentlig ikke endres på? Vil føre til ny versjon uten noen endringer. Samme med README'n til nav-frontend-toggle.

Reply:
Litt usikker hva som er årsaken til at rekkefølgen på disse dependenciene i README-filene blir stokket om på denne måten. Virker å skje ganske regelmessig når vi gjør endringer i `package.json` på rot. Kan ta en titt på `generateReadmes.js` for å se om vi kan tvinge en bestemt rekkefølge på disse så vi unngår slik unødvendig omstokking fremover.